kimchi/src/circuits/gate.rs

```diff
@@ -273,6 +298,24 @@ impl<F: FftField> LookupInfo<F> {
     }
 }
 
+/// Specifies the relative position of gates and the fixed lookup table (if applicable) that a
+/// given lookup configuration should apply to.
+pub struct GatesLookupSpec {
+    /// The set of positions relative to an active gate where a lookup configuration applies.
+    pub gate_positions: HashSet<(GateType, CurrOrNext)>,
+    /// The fixed lookup table that should be used for these lookups, if applicable.
+    pub gate_lookup_table: Option<GateLookupTable>,
+}
```

Comment:
What are cases when there is no fixed lookup table associated?

Reply:
When you're using a runtime or side-loaded lookup table. So none yet, but this is future-proofing.